web/libs/datamanager/src/utils/helpers.ts

```diff
@@ -12,6 +12,35 @@ export const formDataToJPO = (formData: FormData) => {
   return formData;
 };
 
+/**
+ * Hydrate JSON values that are large integers to strings.
+ */
+export const jsonReviverWithBigInt = (_key: any, value: any, context: any) => {
```

Comment:
easy fix for types problem:
```suggestion
export const jsonReviverWithBigInt = (_key: any, value: any, context?: any) => {
```

not all the browsers support context and while your code will work everywhere the typing for `parse()` has no `context` yet, so marking it as an optional will calm TS down.